source/options.js

```diff
@@ -5,10 +5,10 @@ optionsStorage.syncForm('#options-form');
 
 for (const inputElement of document.querySelectorAll('#options-form [name]')) {
 	inputElement.addEventListener('change', () => {
-		// `webext-options-sync` debounces syncing to 100ms, so send updates sometime after that
+		// `webext-options-sync` debounces syncing to 600ms, so send updates sometime after that
 		setTimeout(() => {
 			browser.runtime.sendMessage('update');
-		}, 200);
+		}, 700);
```

Comment:
@fregante, will be looking out for a new release of `webext-options-sync` to get rid of these arbitrary timeout values.

Reply:
Indeed, you can use the version I suggested and then:

```js
select('form').addEventListener('options-sync:form-synced', () => {
	browser.runtime.sendMessage('update');
});

Reply:
Yup, was trying to use that and soon found that this wasn't published to stable yet.

Reply:
> Yup, was trying to use that and soon found that this wasn't published to stable yet.

It will be soon enough, it'd be good to test it here as well.

Reply:
https://github.com/fregante/webext-options-sync/issues/36